mmc/item_updater_mmc.cpp

```diff
@@ -78,6 +78,9 @@ void ItemUpdaterMMC::reset()
     // Set attribute to clear hypervisor NVRAM
     utils::setClearNvram(bus);
 
+    // reset the enabled property of dimms/cpu after factory reset
+    gardReset.reset();
```

Comment:
Please use guard instead of gard. applicable to everywhere in this commit.

Reply:
gardReset variable is existing code, so will not touch that